config/locales/client.en.yml

```diff
@@ -7519,8 +7519,8 @@ en:
         discard: "Discard changes"
         save: "Save all changes"
```

Comment:
Not related to this PR but does it make sense to say "Save change" and `Discard Change` when "You have 1 unsaved change"?